services/addons/images/count_metric/mongo_counter.py

```diff
@@ -6,12 +6,17 @@
 message_types = ["BSM", "TIM", "Map", "SPaT", "SRM", "SSM", "PSM"]
 
 def write_counts(mongo_db, counts):

```

Comment:
I think it will just error out, but since there is no counts it wouldn't have uploaded anything regardless. This just reduces unneeded errors. I just pushed another commit that that will add further error handling if it fails to aggregate on a specific message type.